mycity/mycity/utilities/arcgis_utils.py

```diff
@@ -0,0 +1,283 @@
+import requests
+import json
+import os
+import sys
+import urllib
+import logging
+
+logger = logging.getLogger(__name__)
+
+DRIVING_DISTANCE_TEXT_KEY = "Driving_distance"
+DRIVING_TIME_TEXT_KEY = "Driving_time"
+
+def generate_access_token():
+    """
+    Generates a temporary access token fro ArcGIS REST APIs
+
+    :return: String containing temporary access token
+    """
+
+    try:
```

Comment:
`os.getenv` doesn't throw an exception. So, you can wrap them in functions that will raise `SomeException` if the variable is not found. 

Then you can use one try / except clause  for this function instead of two

Reply:
Each environment variable is now wrapped in its own function, which throws an exception if the variable is not set. The calling function now catches these exceptions